.travis.yml

```diff
@@ -36,16 +36,16 @@ jobs:
         - make .gitvalidation
         - make gofmt
         - make lint
-      go: tip
+      go: "1.10.x"
```

Comment:
Do you want to add a build for tip, even if you don't require it to pass?  FWIW, that's what Buildah does.